spec/fixtures/files/datahashes/botgarden/propagation1.json

```diff
@@ -5,14 +5,14 @@
     "propDate": "2020-11-09",
     "propReason": "conservation",
     "propComments": "Be happy",
-    "scarStrat": "cold strat^^boiling water^^",
-    "duration": "25^^10",
-    "durationUnit": "Hours^^Days^^",
+    "scarStrat": "cold strat;boiling water;",
+    "duration": "25;;10",
+    "durationUnit": "Hours;Days;",
     "extraSeeds": "true",
     "spores": "true",
-    "cuttingType": "hardwood^^",
-    "hormone": "hormone^^",
-    "concentration": "Worry^^",
```

Comment:
These fields are not repeatable, so use no delimiter

Reply:
Noted.